src/EFCore.PG/ValueGeneration/Internal/NpgsqlValueGeneratorCache.cs

```diff
@@ -30,7 +31,9 @@ public NpgsqlValueGeneratorCache([NotNull] ValueGeneratorCacheDependencies depen
         ///     This API supports the Entity Framework Core infrastructure and is not intended to be used
         ///     directly from your code. This API may change or be removed in future releases.
         /// </summary>
-        public virtual NpgsqlSequenceValueGeneratorState GetOrAddSequenceState(IProperty property)
+        public virtual NpgsqlSequenceValueGeneratorState GetOrAddSequenceState(
+            IProperty property,
+            IRelationalConnection connection)
```

Comment:
Does this need a null check added?

Reply:
Same as above, removing null checks for internal type as in upstream.